src/app/components/RecieveTab/index.tsx

```diff
@@ -0,0 +1,66 @@
+import Button from "@components/Button";
+import Container from "@components/Container";
+import DualCurrencyField from "@components/form/DualCurrencyField";
+import TextField from "@components/form/TextField";
+import { useTranslation } from "react-i18next";
+
+type Props = React.FormHTMLAttributes<HTMLFormElement> & {
+  handleSubmit?: (event: React.FormEvent<HTMLFormElement>) => void;
+  loading?: boolean;
+  fiatAmount: string;
+  handleChange?: (
+    event: React.ChangeEvent<HTMLInputElement | HTMLSelectElement>
+  ) => void;
+};
+
+const RecieveTab = ({
```

Comment:
This isn't actually a tab. And it's only for lightning, right?
Should it be called `LightningReceiveForm` or something?